tests/30rooms/01state.pl

```diff
@@ -259,15 +269,14 @@ sub set_test_state
 
    matrix_put_room_state( $user, $room_id,
          type      => "a.test.state.type",
-         state_key => "",
+         state_key => $user->user_id,
```

Comment:
why this change, ooi?

Reply:
Mostly just to match the filter `return unless $event->{type} eq $type and $event->{state_key} eq $user->user_id;` in `sync_timeline_contains`.